### ISSUE TITLE
Scrolling (C-p C-n) over results makes them loose color until the query changes (adding or removing)

### ISSUE THREAD

**Issue by felipesere** [nvim-telescope/telescope.nvim#335]
### Description
When I trigger `find_files` or any other picker, they initially show with colored icons just fine.
When I then scroll to the any item, items I have scrolled over loose their color.

**Expected Behavior**
I expect the the currently selected result to look different, but all other results to use the appropriate colored icons.

**Actual Behavior**
Items scrolled over turn white until changing the query refreshes the screen.

### Details

<details><summary>Reproduce</summary>
1. Open vim
2. :Telescope find_files
3. Ctrl-P a couple of times
4. Bang, colors in the search result are gone at the bottom. (screenshots)

Settings for `telescope.nvim`:

</details>

<details><summary>Environment</summary>

nvim --version
NVIM v0.5.0-nightly-131-g104749440
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/tmp/neovim-20201125-14291-1afp395/build/config -I/tmp/neovim-20201125-14291-1afp395/src -I/usr/local/include -I/tmp/neovim-20201125-14291-1afp395/deps-build/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX11.0.sdk/usr/include -I/usr/local/opt/gettext/include -I/tmp/neovim-20201125-14291-1afp395/build/src/nvim/auto -I/tmp/neovim-20201125-14291-1afp395/build/include
Compiled by felipesere@Felipes-MacBook-Pro.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/HEAD-1047494/share/nvim"

Run :checkhealth for more info

macOS BigSur
❯ git log --pretty=format:'%h' -n 1
9a97982

</details>

<details><summary>Configuration</summary>
<p>
 

```viml
set nocompatible hidden laststatus=2

if !filereadable('/tmp/plug.vim')
  silent !curl --insecure -fLo /tmp/plug.vim
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
endif

source /tmp/plug.vim
call plug#begin('/tmp/plugged')
Plug 'nvim-lua/popup.nvim'
Plug 'nvim-lua/plenary.nvim'
Plug 'nvim-telescope/telescope.nvim'
Plug 'kyazdani42/nvim-web-devicons'
call plug#end()

autocmd VimEnter * PlugClean! | PlugUpdate --sync | close
lua << EOF

require'nvim-web-devicons'.setup {
 -- globally enable default icons (default to false)
 default = true;
}

vim.api.nvim_command("highlight! link TelescopeSelection Type")
vim.api.nvim_command("highlight! link TelescopeMatching Statement")
EOF

nmap <c-p> :execute 'Telescope find_files'<CR>
nmap <silent> <Leader>s :execute 'Telescope grep_string'<CR>
nmap <silent> <Leader>S :execute 'Telescope live_grep'<CR>
```
</details>

<img width="915" alt="Screenshot 2020-12-14 at 08 49 59" src="https://user-images.githubusercontent.com/1850188/102056575-df212680-3dec-11eb-8d68-5f150afe50c9.png">
<img width="905" alt="Screenshot 2020-12-14 at 08 49 48" src="https://user-images.githubusercontent.com/1850188/102056582-e0eaea00-3dec-11eb-9f00-e773a8522d05.png">
<img width="917" alt="Screenshot 2020-12-14 at 08 49 37" src="https://user-images.githubusercontent.com/1850188/102056586-e1838080-3dec-11eb-810d-bb0b15165310.png">

**Comment by Conni2461** [comment]
Duplication of https://github.com/nvim-telescope/telescope.nvim/issues/233

**Comment by Conni2461** [comment]
Fixed in #355
